model/Attachments.inc.php

```diff
@@ -189,15 +189,15 @@ public static function getTemporaryURL(Zotero_Item $item, $localOnly=false) {
 			throw new Exception("Unable to create directory '$downloadDir'");
 		}
 		if ($zip) {
-			$response = Zotero_Storage::downloadFile($info, $downloadDir);
+			$response = Zotero_Storage::downloadFile($info, $downloadDir, '1.zip');
 		}
 		else {
 			$response = Zotero_Storage::downloadFile($info, $downloadDir, $realFilename);
 		}
 		if ($response) {
 			if ($zip) {
-				$success = self::extractZip($downloadDir . $info['filename'], $dir);
```

Comment:
If we create a random dir, I think there is no difference what filename the zip file has.